book/06-github/sections/3-maintaining.asc

```diff
@@ -89,7 +89,7 @@ You could technically merge in the Pull Request work with something like this:
 
 [source,console]
 ----
-$ curl http://github.com/tonychacon/fade/pull/1.patch | git am
+$ curl https://github.com/tonychacon/fade/pull/1.patch | git am
```

Comment:
```
--- ~ » curl https://github.com/tonychacon/fade/pull/1.patch
<html><body>You are being <a href="https://patch-diff.githubusercontent.com/raw/tonychacon/fade/pull/1.patch">redirected</a>.</body></html>%                                                                                                  

--- ~ » curl https://patch-diff.githubusercontent.com/raw/tonychacon/fade/pull/1.patch
From 4276e81262baecc469566147662a0a94a16f6df6 Mon Sep 17 00:00:00 2001
From: Scott Chacon <dragon@github.com>
Date: Thu, 2 Oct 2014 10:05:16 +0200
Subject: [PATCH] wait longer to see the dimming effect better

---
 fade.ino | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fade.ino b/fade.ino
index 599aca1..aec64f5 100644
--- a/fade.ino
+++ b/fade.ino
@@ -21,5 +21,5 @@ void loop()  {
     fadeAmount = -fadeAmount ;
   }
   // wait for 30 milliseconds to see the dimming effect
-  delay(30);
+  delay(40);
 }
```

Yup, totally works.

Reply:
Okay, so the curl part is fine then. :+1: